parlai/tasks/taskmaster/build.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import parlai.core.build_data as build_data
+import os
+
+
+def build(opt):
+    # get path to data directory
+    dpath = os.path.join(opt['datapath'], 'taskmaster-1')
+    # define version if any
+    version = None
```

Comment:
could you please make the version `1.0` here, in case it ever changes

Reply:
Done so in latest commit